### ISSUE TITLE
Support DataBase.check_exists(Model)

### ISSUE THREAD

**Issue by vsmelov** [Infinidat/infi.clickhouse_orm#52]
Sometimes, when I dump some analytical data to ClickHouse I want to be sure, that now there is no such table in DataBase.
Append-mode is not satisfy me
Drop-And-Create too, because I afraid to drop someone important data.

So, now, I need to make workaround
```
    try:
        resp = db.raw('select * from {} limit 0'.format(Model.table_name()))
        assert resp == ''
    except DatabaseException:
        exists = False
    else:
        exists = True
```

May be there is more elegant way to do it?

**Comment by M1hacka** [comment]
A problem, connected to this ticket:
It would be good to parametrize `IF EXISTS` and `IF NOT EXIST` inside `db.create_database` and `db.drop_database` methods. Sometimes, it's important to have strict create and drop operation.
In @vsmelov case it could be also checked by:
`try:
    db.create_database(if_not_exists=False)
    return True
except DatabaseError:
   return False
`

**Comment by ishirav** [comment]
Fixed in v1.0.2
